### DISCUSSION TITLE
Enable/Disable ID/Server

### DISCUSSION THREAD

**Discussion by dragoncity17** [rustdesk/rustdesk#7094] [General]
Hello,
How enable or disable "ID/Server" easier...?

![09-02-2024-Capture038](https://github.com/rustdesk/rustdesk/assets/19295070/73a681ea-5142-435d-a562-9a3847164416)

For the moment I need to remove... and put again... not possible to switch to default/private server ?

Thanks in advance ! :)

**Comment by serg499** [comment]
Please see here (no solution yet): [https://github.com/rustdesk/rustdesk/discussions/6922](https://github.com/rustdesk/rustdesk/discussions/6922)

**Reply by dragoncity17** [reply]
Thank you for your reply.

Ok... but maybe possible to have an "shorcut" rustdesk for private server ..? 
I saw config in user folder (windows).

RustDesk_default.toml
and
RustDesk.toml

C:\Users\"Your Name"\AppData\Roaming\RustDesk\config\RustDesk.toml
